README.md

```diff
@@ -1,63 +1,3280 @@
+    Active: active (running) since Tue 2019-12-03 11:46:10 UTC; 12min ago
+    Main PID: 1448 (openvpn)
+    Memory: 1.3M
+    CGroup: /system.slice/openvpn.service
+            └─1448 /usr/sbin/openvpn --writepid /var/run/openvpn/openvpn.pid --cd /etc/openvpn/ --config /etc/openvpn/openvpn.conf --connect-retry 5 120
+
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /sbin/ip link set dev resin-vpn up mtu 1500
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /sbin/ip addr add dev resin-vpn local 10.240.53.172 peer 52.4.252.97
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /etc/openvpn-misc/upscript.sh resin-vpn 1500 1555 10.240.53.172 52.4.252.97 init
+    Dec 03 11:46:13 11a12ec openvpn[1448]: resin-ntp-config: Found config.json in /mnt/boot/config.json .
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 /sbin/ip route add 52.4.252.97/32 via 52.4.252.97
+    Dec 03 11:46:13 11a12ec openvpn[1448]: ip: RTNETLINK answers: File exists
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 ERROR: Linux route add command failed: external program exited with error status: 2
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 GID set to openvpn
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 UID set to openvpn
+    Dec 03 11:46:13 11a12ec openvpn[1448]: Tue Dec  3 11:46:13 2019 Initialization Sequence Completed
+    ```
+* `systemctl start <serviceName>` - Will start a non-running service. Note that
+    this will *not* restart a service that is already running.
+* `systemctl stop <serviceName>` - Will stop a running service. If the service
+    is not running, this command will not do anything.
+* `systemctl restart <serviceName>` - Will restart a running service. If the
+    service is not running, this will start it.
+* `systemctl daemon-reload` - Will essentially run through the startup process
+    systemd carries out at initialisation, but without restarting services or
+    units. This allows the rebuild of the system dependency graph, and should be
+    carried out if any of the unit files are changed whilst the system is
+    running.
+
+This last command may sound a bit confusing, but take a quick example. Imagine
+that you need to dynamically change the `resin-supervisor.service` unit file
+for some reason (maybe to increase the healthcheck timeout on a very slow
+system). Once that change has made, you'll want to restart the service. However,
+first, you need to reload the unit file as this is changed from the loaded
+version. To do this, you'll run `systemctl daemon-reload` and then
+`systemctl restart resin-supervisor.service` to restart the Supervisor.
+
+In general, there are some core services that need to execute for a device to
+come online, connect to the balenaCloud VPN, download applications and then run
+them:
+
+* `chronyd.service` - Responsible for NTP duties and syncing 'real' network
```

Comment:
consider also listing `timesyncd` which was used in older versions?

Reply:
Done.